README.md

```diff
@@ -78,29 +78,11 @@ path = "fake_lib.rs"
 [dependencies]
 log = "=0.3.6"
 
-[workspace.metadata.raze]
```

Comment:
While the documentation here is extremely similar, the difference is `[workspace.metadata` vs `[package.metadata`. I think it's worth adding a comment explaining that there are two settings

Reply:
but don't feel super strongly about it 😄 Overall still an improvement in my eyes

Reply:
This example is a package-level `Cargo.toml` (line 70 is `[package]`) rather than a workspace-level one, right? Do we use `workspace.metadata` even for a single-package workspace (i.e. when it's just a `[package]`)?

There's a later section on "Cargo workspace projects" that says "use `workspace.metadata.raze`"; I think that covers the workspace case.

Reply:
Good call. You **can** use workspace metadata for single packages. Cargo metadata always contains some notion of a workspace even if you only have a single root package. That said though, it'd be more consistent to use the package's metadata for single package crates (or at least that's what I'd expect to see as a reader). So this looks good to me 😄

Reply:
@UebelAndre Do you think there's value in having these `[workspace]` instructions elsewhere?

Reply:
That should be covered by https://github.com/google/cargo-raze#cargo-workspace-projects (which I forgot existed). That's what @cceckman was mentioning earlier in this thread. 

Though, in reading the docs, I see the rendering is bad (my fault 😅 if there's bandwidth to fix that here it'd be greatly appreciated but I can otherwise open another PR)
<img width="851" alt="Screen Shot 2021-01-05 at 7 15 48 PM" src="https://user-images.githubusercontent.com/26427366/103725029-8deaf980-4f8a-11eb-89c5-31c4723d5cab.png">

I think this is otherwise a good improvement.

Reply:
Went ahead and fixed the rendering:

![Screenshot 2021-01-10 at 10 47 15](https://user-images.githubusercontent.com/861696/104132306-5afd8880-5331-11eb-9ede-113400acdbcf.png)